pallets/vc-management/src/assertion.rs

```diff
@@ -20,26 +20,26 @@
 // See: https://www.notion.so/litentry/Expected-parameters-in-predefined-rulesets-14f74928aa2b43509167da12a3e75507
 
 use codec::{Decode, Encode, MaxEncodedLen};
-use primitives::Balance;
 use scale_info::TypeInfo;
 use sp_runtime::{traits::ConstU32, BoundedVec};
 
+type Balance = u128;
```

Comment:
Any reasons not to use primitives::Balance?

Reply:
Strange. Didn't noticed this change. Will check.